src/PerfView/StackViewer/StackWindow.xaml.cs

```diff
@@ -516,6 +516,7 @@ private void FixupJustMyCodeInGroupPats(StackSource stackSource)
                 }
 
                 GroupRegExTextBox.Text = justMyApp;
+                GroupRegExTextBox.Items.Add(@"[join all threads]            Thread -> AllThreads");
```

Comment:
Why do you need to add the item twice?  The first one in PerfViewData.cs I understand, but I'm not clear on why you have to add it here as well.